src/components/Switcher/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+import Image from 'next/image';
+import Style from './ThemeSwitcher.module.scss';
+import { useTheme } from '@/contexts/ThemeContext/ThemeContext';
+
+export default function ThemeSwitcher() {
+  const { theme, setNewTheme } = useTheme();
+  return (
+    <div className={Style.switcher}>
+      <button className={Style.switcher__button} onClick={() => setNewTheme('light')}>
+        <Image
+          className={
+            theme === 'light'
+              ? `${Style['switcher__icon']} ${Style['switcher__icon-selected']}`
+              : Style['switcher__icon']
+          }
+          src={'/sun.svg'}
+          alt='sun'
+          width={30}
+          height={30}
+        />
+      </button>
+      <button className={Style.switcher__button} onClick={() => setNewTheme('auto')}>
```

Comment:
Please remove the third button, the theme toggle should have only 2 buttons or you can make it with 1 button which will toggle theme no matter on which icon you click